config/locales/zh-TW.yml

```
@@ -175,32 +175,32 @@ zh-TW:
       title: 修改密碼
       will_email_notice: 系統將會寄一封包含重設密碼連結的電子郵件給你
     otp_prompt:
-      authenticate:
+      authenticate: 驗證
   multifactor_auths:
-    incorrect_otp:
-    otp_code:
-    require_mfa_disabled:
-    require_mfa_enabled:
+    incorrect_otp: 你的 OTP 碼 不正確。
+    otp_code: OTP 碼
+    require_mfa_disabled: 你的多重要素驗證已啟用，請先停用。
+    require_mfa_enabled: 你的多重要素驗證已停用，請先啟用。
     new:
-      title:
-      scan_prompt:
-      otp_prompt:
-      confirm:
-      enable:
-      account:
-      key:
-      time_based:
+      title: 啟用多重要素驗證
+      scan_prompt: 請用你的驗證裝置掃描 QR-code。如果你沒辦法掃描，手動輸入下面的資料。
+      otp_prompt: 輸入驗證裝置上的數字來繼續。
+      confirm: 我已把復原碼收在安全的地方。
+      enable: 啟用
+      account: "帳號: %{account}"
+      key: "金鑰: %{key}"
+      time_based: "Time based: 是"
     create:
-      qrcode_expired:
-      success:
+      qrcode_expired: QR-code 和金鑰已過期。請再重新註冊裝置。
+      success: 你已成功啟用多重要素驗證。
     recovery:
-      continue:
-      title:
-      note:
+      continue: 繼續
+      title: 復原碼
+      note: 你必須要把復原碼收在安全的地方來避免失去帳號。失去認證裝置的情況下，每一個復原碼可以使用一次。
```

I would say "請務必將復原碼收在安全的地方以避免失去帳號"
```suggestion
      note: 請務必將復原碼收在安全的地方以避免失去帳號。失去認證裝置的情況下，每一個復原碼可以使用一次。
```